src/data/thunks.js

```diff
@@ -237,7 +237,7 @@ export function pollAttempt(url) {
         ...currentAttempt,
         time_remaining_seconds: data.time_remaining_seconds,
         accessibility_time_string: data.accessibility_time_string,
-        attempt_status: data.status,
+        attempt_status: data.attempt_status,
```

Comment:
this change isn't going to be backwards compatible with edx-proctoring. If you need to update the dictionary key, I would do that in api.js where you know if you are calling edx-exams